tests/bsim/bluetooth/host/privacy/peripheral/src/tester.c

```diff
@@ -11,53 +11,72 @@
 #include <zephyr/bluetooth/bluetooth.h>
 
 #define EXPECTED_NUM_ROTATIONS 5
+#define MANUFACTURING_DATA_LEN 1
 
-static bt_addr_le_t old_addr;
-static int64_t old_time;
-static int rpa_rotations;
+struct adv_set_data_t {
+	bt_addr_le_t	old_addr;
+	int64_t			old_time;
+	int				rpa_rotations;
+};
+
+char	bt_data[MANUFACTURING_DATA_LEN];
```

Comment:
`bt_data` is too generic a name here IMO, and somewhat conflicts with the `struct bt_data`. Suggest to rename and also make `static`

Reply:
Renamed.